core/src/main/thrift/client.thrift

```diff
@@ -46,13 +46,13 @@ enum TableOperationExceptionType {
   NOTFOUND
   OFFLINE
   BULK_BAD_INPUT_DIRECTORY
-  BULK_BAD_ERROR_DIRECTORY
+  OBSOLETE_BULK_BAD_ERROR_DIRECTORY
```

Comment:
If we've already changed the thrift RPCs substantially so that they are already incompatible in other ways, then this rename probably doesn't matter. I'm not sure if that's the case or not.

Reply:
I don't know if we have changed them substantially in 3.0. Maybe a follow-on task to evaluate the Thrift API changes before 3.0 release?